src/modules/developer_tools.json

```diff
@@ -0,0 +1,250 @@
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default Example",
+                    "params": [],
+                    "result": {
+                        "name": "Default Result",
+                        "value": null
+                    }
+                }
+            ]
+        },
+        {
+            "name": "pair",
+            "summary": "Configures Firebolt to accept incoming Firebolt JSON RPC messages from the developer tool identified by the given developer tool ID",
+            "tags": [],
+            "params": [
+                {
+                    "name": "developerToolId",
+                    "summary": "The ID of the DeveloperTool to which to connect",
+                    "required": true,
+                    "schema": {
+                        "type": "string",
+                        "format": "uuid"
+                    }
+                }
+            ],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "const": null
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default Example",
+                    "params": [
+                        {
+                            "name": "developerToolId",
+                            "value": "ff6340fb-8576-494c-97c6-e60e9a4eeba9"
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": null
+                    }
+                }
+            ]
+        },
+        {
+            "name": "unpair",
+            "summary": "Disconnect Firebolt as a web socket client from the web socket that corresponds to the given developer tool ID",
+            "tags": [],
+            "params": [
+                {
+                    "name": "developerToolId",
+                    "summary": "The ID of the DeveloperTool from which to disconnect",
+                    "required": true,
+                    "schema": {
+                        "type": "string",
+                        "format": "uuid"
+                    }
+                }
+            ],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "const": null
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default Example",
+                    "params": [
+                        {
+                            "name": "developerToolId",
+                            "value": "ff6340fb-8576-494c-97c6-e60e9a4eeba9"
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": null
+                    }
+                }
+            ]
+        },
+        {
+            "name": "onDeveloperToolUp",
```

Comment:
We try to be a bit more formal with the API names. "Up"/"Down" is a bit colloquial.

Propose: "Available" / "Unavailable"

Also propose shortening name, e.g. `onToolAvailable`

Reply:
named 'onToolAvailable' and 'onToolUnavailable' per your suggestion